packages/transformer/src/test/TestUtils/TimelineTestUtil.ts

```diff
@@ -251,6 +252,10 @@ export type TimelineStateChange =
           since?: number;
           initTransformer?: (transformer: IModelTransformer) => void;
           expectThrow?: boolean;
+          assertStatePostProcessChanges?: (
+            transformer: IModelTransformer
+          ) => void;
```

Comment:
what about a new subobject?
```suggestion
          assert?: {
            afterProcessChanges?(transformer: IModelTransformer): void;
          };
```